drivers/pulse_counter/include/pulse_counter_params.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2017 UC Berkeley
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_pulse_counter
+ *
+ * @{
+ * @file
+ * @brief       Default configuration for PULSE_COUNTER devices
+ *
+ * @author      Hyung-Sin Kim <hs.kim@cs.berkeley.edu>
+ */
+
+#ifndef PULSE_COUNTER_PARAMS_H
+#define PULSE_COUNTER_PARAMS_H
+
+#include "board.h"
+#include "pulse_counter.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   PULSE_COUNTER configuration
+ */
+static const pulse_counter_params_t pulse_counter_params[] =
```

Comment:
Please also add a default configuration (as e.g. discussed in #7519 and applied in e.g. #7955 or #7949) -> this makes it much easier for the CI to build this driver even without board specific configuration.

Reply:
sure, thanks :)